blog/2020-11-26-project-waves.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Project Waves: Swapping Sats on Liquid"
+author: philipp
+author_url: https://github.com/bonomat
+author_image_url: https://avatars2.githubusercontent.com/u/224613
+tags: [dev-update,liquid,elementsproject]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img alt="Waves" src={useBaseUrl('blog/assets/images/2020-11/btc_waves.png')} />
+
+> "You can't stop the waves, but you can learn to surf.", Jon Kabat-Zinn
+
+COMIT's vision is to have an open financial system - truly inclusive, censorship resistant and non-discriminatory. 
+And we believe that Bitcoin is the most promising platform for such an open financial system due to its decentralization and its censorship resistance. 
+
+Ultimately we would love to see such an open financial system be built on top of Bitcoin, however, as of today, Bitcoin lacks privacy and only supports a single asset (BTC).
+
+As mentioned in [Project Droplet](https://medium.com/comit-network/project-droplet-atomic-swaps-on-liquid-4c5ac045ad3c), we believe that Liquid can be a temporary platform allowing us to build a financial ecosystem around Bitcoin. 
```

Comment:
Like I said in the review of the Tweet thread, I don't really like this way of putting it. Temporary has such a bad co-notation where we are only doing this as a workaround and not because we think it is actually a good solution.

I know that from a certain perspective it is true! However, instead of using "temporary" I'd frame it more positively by saying: "We are building this on top of what we think is the best solution as of today."